packages/instant-meilisearch/src/adapter/search-request-adapter/search-params-adapter.ts

```diff
@@ -232,6 +232,12 @@ export function MeiliParamsCreator(searchContext: SearchContext) {
         meiliSearchParams.attributesToSearchOn = value
       }
     },
+    addHybridSearch() {
```

Comment:
Can you add a test case for that?

Reply:
I added a test! It seems to work, but maybe there's a different way to approach this. Let me know!